spec/features/pilot/step_1_understand_service_spec.rb

```diff
@@ -26,6 +27,18 @@ def should_see_what_the_service_can_provide
     should_see_list_of_landable_bodies
   end
 
+  def should_see_what_information_i_will_need_to_provide
+    should_see_that_i_will_need_to_provide("name")
+    should_see_that_i_will_need_to_provide("date of birth")
+    should_see_that_i_will_need_to_provide("the dates of your proposed visit")
+    should_see_that_i_will_need_to_provide("passport number")
+    should_see_that_i_will_need_to_provide("email")
+    should_see_that_i_will_need_to_provide("pilot's licence number")
+    should_see_that_i_will_need_to_provide("vehicle registration number")
```

Comment:
This pattern seems to contrast with the loop approach in `should_see_list_of_landable_bodies` - is there a reason for using a loop over an array of strings in one and a repeated method call with varying argument in the other?

Reply:
This was actually deliberate, to illustrate different approaches -- which might be more readable depending on the context#1